package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@openst/facilitator",
+  "description": "Facilitator is an executable to process stake & mint and redeem & unstake operations.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/OpenST/facilitator.git"
+  },
+  "keywords": [
+    "Facilitator",
+    "OpenST",
+    "OST",
+    "Mosaic",
+    "Simple Token",
+    "Token Economy",
+    "web3",
+    "Ethereum",
+    "EIP20"
+  ],
+  "homepage": "https://openst.org",
+  "author": "OpenST Ltd.",
+  "license": "LGPL-3.0-only",
+  "bugs": {
+    "url": "https://github.com/openst/mosaic.js/issues"
+  },
+  "scripts": {
+    "lint": "eslint src -c .eslintrc.json --ext ts"
+  },
+  "dependencies": {
+    "typescript": "^3.3.3333"
```

```suggestion
    "typescript": "3.3.3333"
```
Is it a dev dependency?

---

Not 100% sure. In mosaic chains also, it's in dependency.